### ISSUE TITLE
Add option to output TIFF files for publications

### ISSUE THREAD

**Issue by atruskie** [QutEcoacoustics/audio-analysis#182]
### Expected behaviour

A user should be able to enable outputting all images as TIFF files for publications.

### Actual behaviour

AP.exe only outputs PNG image files

### Any other details

This is a low priority feature. We'll probably do it after we've transitioned to ImageSharp (#158 )

**Comment by stewartmacdonald** [comment]
PNG is lossless, right? What advantages are there to exporting as TIFF? Just compatibility with publishers that require TIFF?

**Comment by atruskie** [comment]
You're correct. This is purely just a convenience feature requested by an ecologist.
